lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -14,10 +14,9 @@
 
   def self.instances
     PuppetX::PuppetLabs::ScheduledTask::V2Adapter.tasks.collect do |job_file|
```

Comment:
Can probably rename the iterator here as `task` as job_file has no meaning in the V2 land

Reply:
Done